### PR TITLE
[api] adding importer_submit api to public operation

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- adding importer_submit_api to public operation

### CODE REVIEW

File: desktop/core/src/desktop/api_public_urls.py [hunk 1]
```diff
@@ -104,6 +104,7 @@
 urlpatterns += [
   re_path(r'^indexer/guess_format/?$', api_public.guess_format, name='indexer_guess_format'),
   re_path(r'^indexer/guess_field_types/?$', api_public.guess_field_types, name='indexer_guess_field_types'),
+  re_path(r'^importer/submit', api_public.importer_submit, name='indexer_importer_submit'),
```
Comment (Harshg999) [inline desktop/core/src/desktop/api_public_urls.py:107]: Maybe `indexer/importer/submit` to keep the pattern? And same update in docs too?

**Review by romainr — COMMENTED** [review]
:+1: On Harsh's comment

File: desktop/core/src/desktop/api_public_urls.py (hunk 1 above)
Comment (agl29) [reply desktop/core/src/desktop/api_public_urls.py]: done

File: docs/docs-site/content/developer/api/rest/_index.md [hunk 2]
```diff
@@ -458,7 +458,7 @@ Then getting some data sample as well as the column types (column names will be
       .....
     }
 
-Then we submit via `https://demo.gethue.com/indexer/api/importer/submit` and provide the `source` and `destination` parameters. We get back an `operation id` (i.e. some SQL Editor query history id).
+Then we submit via `https://demo.gethue.com/api/indexer/importer/submit` and provide the `source` and `destination` parameters. We get back an `operation id` (i.e. some SQL Editor query history id).
 
 If the `show_command` parameter is given, the API call will instead return the generated SQL queries that will import the data.
```
Comment (romainr) [inline docs/docs-site/content/developer/api/rest/_index.md:463]: Maybe it is time to add a complete example of the request data? (and truncate/remote the fields actually not used in submit to simplify to the maximum)
(as the API is close to impossible to use if we don't document those)

**Review by romainr — APPROVED** [review]

**Pull request merged**